README.md

```diff
@@ -25,18 +25,16 @@ Check out the [examples](examples) directory.
 
 1. Install the library
 
-        git clone https://github.com/jupyter-incubator/sparkmagic
-        cd sparkmagic
-        pip install -e .
+        pip install sparkmagic
 
 2. (Optional) Install the wrapper kernels
 
-        jupyter-kernelspec install remotespark/kernels/sparkkernel
-        jupyter-kernelspec install remotespark/kernels/pysparkkernel
+        jupyter-kernelspec install sparkmagic/kernels/sparkkernel
+        jupyter-kernelspec install sparkmagic/kernels/pysparkkernel
         
-3. (Optional) Copy the example configuration file to your home directory 
+3. (Optional) Copy the example configuration file to your home directory. Look at the example config at https://github.com/jupyter-incubator/hdijupyterutils
```

Comment:
1. You would make this a link directly to the example config, not just to the top level of the repo

2. Obviously, long term, the example config should be in this repo, not that one